core-services/supplemental-ci-images/boilerplate.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  name: boilerplate
+  namespace: openshift
+spec:
+  tags:
+  - annotations:
+      openshift.io/imported-from: quay.io/app-sre/boilerplate:image-v0.1.0
+      description: "Base image used for prow tests by consumers of https://github.com/openshift/boilerplate"
```

Comment:
This description would make more sense in the top-level metadata (under L3). But I don't know whether that will make it show up in the right place(s) on the resulting Image artifacts. Advice?